README.md

```diff
@@ -1,2 +1,47 @@
-# IXI025
-Head dataset based on IXI subject 025, including whole head segmentation, anatomical images, diffusion weighted images
+[![license: CC BY-SA 3.0](https://licensebuttons.net/l/by-sa/3.0/80x15.png)](https://creativecommons.org/licenses/by-sa/3.0/)
+[![downloads](https://img.shields.io/github/downloads/ITISFoundation/IXI025/total.svg)](<>)
+
+# IXI025 Model - IT'IS Template Head Segmentation Repository
+
+The IXI025 head model is based on Subject 025 from the [IXI dataset](https://brain-development.org/ixi-dataset/). It includes a whole head segmentation, surface-based model, fiducials for 10-10 system construction, as well as the original anatomical (T1w, T2w, PD, MRA) and diffusion weighted images. The head segmentation/model is in the original scanner coordinates, i.e., it is perfectly aligned with the magnetic resonance images.
+
+This dataset is made public to support research in computational brain stimulation modeling, e.g., transcranial electric, magnetic, and acoustic brain stimulation, as well as temporal interference (TI) stimulation. The head model is used for Sim4Life tutorials by IT'IS and its partner organization ZMT Zurich MedTech AG.
+
+![Head segmentation and model](https://raw.githubusercontent.com/ITISFoundation/IXI025/main/visuals/head_model.png)
```

Comment:
``![Head segmentation and model](visuals/head_model.png)`` does not work?

Reply:
I guess it needs to be merged first.

Reply:
> `![Head segmentation and model](visuals/head_model.png)` does not work?

It does/would! Will fix.